src/pages/project/[uuid]/tabs/Overview.tsx

```diff
@@ -134,14 +93,121 @@ const ProjectOverviewTab = ({ project }: ProjectOverviewTabProps) => {
                 variant="secondary"
                 href={`/entity/sites/create/${project.framework_uuid}?parent_name=projects&parent_uuid=${project.uuid}`}
               >
-                {t("Add Site")}
+                {t("expand view")}
               </Button>
             }
           >
             <ProjectArea project={project} />
           </PageCard>
         </PageColumn>
       </PageRow>
+
+      <PageRow>
+        <PageColumn>
+          <PageCard
+            title={t("Project Monitoring")}
+            tooltip="Lorem ipsum dolor sit amet, urna neque viverra justo nec ultrices dui sapien eget mi proin sed libero."
```

Comment:
Let's make sure no mocked / fake data make it into staging.

Reply:
Done